### ISSUE TITLE
[Beta 1.5.0.4] MCH Flamethrower is not attributed to MCH DPS. 

### ISSUE THREAD

**Issue by Soliel** [ravahn/FFXIV_ACT_Plugin#75]
When used, Flamethrower shows up as an Unknown Dot Tick. On the target's incoming damage list but is nowhere to be seen on the outgoing damage. 

`EncId,STime,Attacker,SwingType,AttackType,DamageType,Victim,Damage,DamageString,Critical,CriticalStr,Special,DmgAdjust,BuffDuration,DmgReduced,OverHeal,DirectHit
423b0f96,2017-06-20 18:14:33,Unknown,20,Unknown Dot Tick,,Striking Dummy,863,863,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:25,Unknown,20,Unknown Dot Tick,,Striking Dummy,877,877,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:27,Unknown,20,Unknown Dot Tick,,Striking Dummy,883,883,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:24,Unknown,20,Unknown Dot Tick,,Striking Dummy,895,895,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:30,Unknown,20,Unknown Dot Tick,,Striking Dummy,908,908,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:28,Unknown,20,Unknown Dot Tick,,Striking Dummy,913,913,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:34,Unknown,20,Unknown Dot Tick,,Striking Dummy,914,914,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:29,Unknown,20,Unknown Dot Tick,,Striking Dummy,922,922,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:31,Unknown,20,Unknown Dot Tick,,Striking Dummy,933,933,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:32,Unknown,20,Unknown Dot Tick,,Striking Dummy,935,935,F,None,None,,,0,0,
423b0f96,2017-06-20 18:14:26,Unknown,20,Unknown Dot Tick,,Striking Dummy,940,940,F,None,None,,,0,0,
`

**Comment by ravahn** [comment]
This turns out to be a challenge for anything other than the simplest fight with one dot.  Flamethrower creates a second independent DoT tick on the affected targets, but doesnt put any debuff or anything on them.  Just causes damage.  And, unlike Shadow Flare and Wildfire, it doesnt include the Buff ID on this unique set of ticks.

There's nothing that I see to distinguish between the combined DoT tick and the flamethrower DoT ticks.

I'm going to have to do some digging on this to see if there is some extra data I can find in the network packets.  I'm not sure how I can even tell which enemies were hit if I can't match up the 'unknown dot tick' damage to the expected DoT damage.

**Comment by Soliel** [comment]
That's something I noticed in game too, not only with this, but with Rook Overdrive/Bishop Overdrive. Both of these skills(While being bad in some situations) Always feel bad to use because the game doesn't attribute the numbers to me, so I don't see them in big flashy text. That being said, I did feel somewhat stupid after I posted this because it seems like something you were already KEENLY aware of. (I didn't know at the time that this specific version didn't parse DoTs Whoops!)

I do appreciate the informative comment and not just immediately dismissing my issue as known.

**Comment by ravahn** [comment]
So, this is somewhat confusing but it turns out the buff ID is actually sent for Flamethrower when the damage hits, similar to wildfire and ground AOEs.  I'm not sure why the above list of skills was missing the Buff ID, and why I didn't notice when I reviewed the logs.

I did find an unrelated bug which explains why when I originally tested this, it didn't immediately work, and I decided to file it away to research later.  It's working now, so this will be fixed in the next version of the plugin.
